### DISCUSSION TITLE
Consider the possibility to add icons to the folders

### DISCUSSION THREAD

**Discussion by sgiraz** [massCodeIO/massCode#209] [Ideas]
Hi guys,

It'd be nice to have the possibility to associate a default(based on the folder name)/custom icons to the folders of our snippets, wouldn't it?

Here is an example:

![image](https://user-images.githubusercontent.com/18039045/183623921-89232915-6e36-40da-af72-94ab3744e384.png)

**Comment by antonreshetov** [comment]
I even have where it is in the backlog ) ) Still thinking ))

**Reply by Benbinbin** [reply]
I think this [component](https://content-wind.nuxt.dev/#icons) in Content Wind (which is a template of Nuxt Content v2) can be a great example to integrate [iconify](https://iconify.design/) (a website collects a huge amount of free icons) into Vue project.

And maybe the most useful icons subset which contains a lot of program language logo is called [SVG Logos](https://icones.js.org/collection/logos)

**Reply by antonreshetov** [reply]
The project already uses https://github.com/antfu/unplugin-icons which can so access Iconify. 

Has not yet made a decision on the implementation of this function

**Reply by Benbinbin** [reply]
Great! I think adding the relative icon for the program language at the popup list maybe also great. 

![Snipaste_2022-08-11_15-07-53](https://user-images.githubusercontent.com/40909550/184081222-6e13828a-9127-4657-94b8-54df35140703.png)
